freqtrade/configuration/deploy_config.py

```diff
@@ -111,6 +111,7 @@ def ask_user_config() -> dict[str, Any]:
                 "binance",
                 "binanceus",
                 "bingx",
+                "bitget",
```

Comment:
we're only adding supported exchanges here - but i don't see bitget as becoming an officially supported exchange at this point (essentially that assumes that core team members have an account on that exchange and are able to actively support people with it).

Reply:
get it, i will remove it